Cargo.lock

```diff
@@ -779,7 +780,7 @@ version = "0.1.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "b8d976903543e0c48546a91908f21588a680a8c8f984df9a5d69feccb2b2a211"
 dependencies = [
- "cfg-if",
+ "cfg-if 0.1.10",
```

Comment:
Which version have we used before? How to guarantee it does not break anything? Here and everywhere.

Reply:
```zsh
doasync@macbook pos-network-node % cargo update --package ring
    Updating crates.io index
    Updating cc v1.0.58 -> v1.0.68
    Updating once_cell v1.4.0 -> v1.8.0
    Updating ring v0.16.15 -> v0.16.20
doasync@macbook pos-network-node % cargo update --package environmental
    Updating crates.io index
    Updating environmental v1.1.2 -> v1.1.3
doasync@macbook pos-network-node % cargo update --package librocksdb-sys
    Updating crates.io index
    Updating bindgen v0.54.0 -> v0.57.0
      Adding cfg-if v1.0.0
    Updating clang-sys v0.29.3 -> v1.2.0
      Adding libloading v0.7.0
    Updating librocksdb-sys v6.11.4 -> v6.17.3
doasync@macbook pos-network-node % cargo update --package fs-swap
    Updating crates.io index
    Updating fs-swap v0.2.4 -> v0.2.6
    Updating libc v0.2.73 -> v0.2.97
```

Reply:
All versions have been updated with no breaking changes according to semver, except for clang-sys.

Reply:
According to `clang-sys` [changelog](https://github.com/KyleMayes/clang-sys/blob/master/CHANGELOG.md) there are no breaking changes from v0.29.3 -> v1.2.0.
Interestingly, the cargo detected that somehow.